doc/contribute/contributor_expectations.rst

```diff
@@ -370,9 +370,9 @@ Reviewer Expectations
 
 - Style changes that the reviewer disagrees with but that are not documented as
   part of the project can be pointed out as non-blocking, but cannot constitute
-  a reason for a request for changes. The reviewer can optionally correct any
-  potential inconsistencies in the tree, document the new guidelines or rules,
-  and then enforce them as part of the review.
+  a reason for a request for changes. The reviewer can optionally propose and
+  document new guidelines or rules, and then enforce them after they've been
+  evaluated, reviewed, agreed and merged.
```

Comment:
Question: wouldn't it be better to move the clause to the section starting at line 377?
Since we point there that reviewers should be able to list out the rules/guidelines, we may also mention that if there is no such rule then reviewer is allowed to form one? Any opinions?

Reply:
Agreed, these two consecutive items seem redundant with each other.

@de-nordic at this point you seem to know much better than me; would you mind submitting something yourself? I'd be more than happy to abandon this one.

Reply:
Thanks for trying. I will try my best to provide alt PR to the end of week.

Reply:
@de-nordic any luck?